### PR TITLE
MS-07 Psychologists - Create Psychologists

### PR DESCRIPTION
**Pull request by JamirMansilla**
##Commits:

- MS-07: fill json with random data
- MS-07: add psychologist functions in psychologist.js
- MS-07: add endpoint to index.js

### CODE REVIEW

File: data/psychologists.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "timeRange":"20:22",
+        "toTimeRange":"8:03",
+        "dayRange":"21/12/2020"
+    },
+    {
+        "id":3,
+        "name":"Nikolia Scinelli",
+        "email":"nscinelli2@pbs.org",
+        "username":"nscinelli2","password":"PbMGLQ9v","phoneNumber":"5595603806","enrollmentNumber":6813,"status":false,"timeRange":"3:00","toTimeRange":"2:36","dayRange":"30/06/2021"
+    },
```
Comment (nicolasblasco) [inline data/psychologists.json:33]: @JamirMansilla fix identation of mock data of psychologist with id: 3

File: data/psychologists.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+[
+    {
+        "id":1,
```
Comment (nicolasblasco) [inline data/psychologists.json:3]: @JamirMansilla change id type to string. Do this for all objects of the mock data.

File: data/psychologists.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        "name":"Clim Jaulme",
+        "email":"cjaulme0@msu.edu",
+        "username":"cjaulme0",
+        "password":"ukWx18Mib",
+        "phoneNumber":"5666233289",
+        "enrollmentNumber":6603,
```
Comment (nicolasblasco) [inline data/psychologists.json:9]: @JamirMansilla change enrollmentNumber type to string. Do this for all objects of the mock data. 
To see which data type goes for each property, see the excel "Recursos"

**Review by damianalvarez — CHANGES_REQUESTED** [review]
@JamirMansilla let's solve the comments that @nicolasblasco left before merging this one

File: controllers/psychologists.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const psychologists = JSON.parse(fs.readFileSync('./data/psychologists.json'));
+
+//MS 07 Create Psychologists
+
+const create = (req, res) =>{
+    const newPsychologist = {
+        id: req.query.id,
```
Comment (damianalvarez) [inline controllers/psychologists.js:8]: @JamirMansilla let's get rid of this property, the ID will be auto-assigned by the DB engine

**Review by damianalvarez — CHANGES_REQUESTED** [review]

File: controllers/psychologists.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const psychologists = JSON.parse(fs.readFileSync('./data/psychologists.json'));
+
+//MS 07 Create Psychologists
+
+const create = (req, res) =>{
+    const newPsychologist = {
+        name: req.query.name,
```
Comment (BaltazarCortina) [inline controllers/psychologists.js:8]: @JamirMansilla we should also assign an id to every new psychologist.

File: index.js [hunk 6]
```diff
@@ -9,10 +10,12 @@ const sessionsController = require('./controllers/sessions.js');
 const profilesController = require('./controllers/profiles.js');
 
 app.set('json spaces', 2);
-
 app.get('/', (req, res) => {
   res.send('Welcome to MindSet!');
 });
+app.listen(port, () => {
+  console.log(`MindSet server listening at http://localhost:${port}`);
+});
```
Comment (BaltazarCortina) [inline index.js:18]: @JamirMansilla These lines are already at the end, they shouldn't be repeated here.

File: index.js [hunk 7]
```diff
@@ -52,6 +55,10 @@ app.get('/sessions/update/:id', sessionsController.update);
 app.get('/profiles', profilesController.getAll);
 app.get('/profiles/create', profilesController.create);
 
+//PSYCHOLOGISTS
+app.get('/psychologists', psychologistController.create);
+
 app.listen(port, () => {
   console.log(`MindSet server listening at http://localhost:${port}`);
 });
+
```
Comment (BaltazarCortina) [inline index.js:64]: @JamirMansilla can we get rid of the empty space in the last line?

**Review by BaltazarCortina — CHANGES_REQUESTED** [review]

**Review by BaltazarCortina — APPROVED** [review]

**Review by damianalvarez — APPROVED** [review]

**Pull request merged**